source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-open-distro.rst

```diff
@@ -292,7 +292,7 @@ Upgrading Kibana
                 .. code-block:: console
 
                   # mkdir -p /usr/share/kibana/data/wazuh/config/
-                  # cp /usr/share/kibana/optimize/wazuh/config/wazuh.yml /usr/share/kibana/data/wazuh/config/wazuh.yml
```

Comment:
This path should not be changed. In this step, the old configuration file is copied to the new location during the upgrade process.

Reply:
Indeed, as you mention both changes were wrong. I have already modified them again. Thanks!